python/tvm/relay/frontend/tflite.py

```diff
@@ -1911,6 +1911,14 @@ def convert_fully_connected(self, op):
             weight_expr = self.exp_tab.new_const(weight_value, dtype=weight_tensor_type_str)
         weight_shape = _infer_shape(weight_expr)
 
+        # Change the output shape calculation based on keep_dim option
```

Comment:
Can you add a test in `tests/python/frontend/tflite/test_forward.py`?

Reply:
The fact is that I can not new a fully_connected op with keep_dim == True or find a model which fully_connected op with keep_dim == True. 
Also, I have tried to construct a matmul in tensorflow which can be converted to fullyconnected by tf.lite.TFLiteConverter. But, I just get the FlexMatmul op in TFLite.
Now, i am stuck。
BTW, I implement it according to [https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/fully_connected.cc](url) line:353

Reply:
Maybe try converting from keras?

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense

Its documentated behavior seems to match keep_dims = True.

I would make just a new unit test if you go this route.

Reply:
As the document said, I constructed the keras model with tf.keras.Input(shape=(1, 1, 768)) and tf.keras.layers.Dense(1001).
![code](https://user-images.githubusercontent.com/32191045/149277656-43fa4989-d665-4b7a-b21f-3e12cb02637b.png)
And then i converted it to TFLite but got a fully connected op(keep_num_dims : False). 
The specific implment in TFLite is to reshape into 2 dims before fullyconnect, and then reshape the output into 4dims.
![fullyconnected](https://user-images.githubusercontent.com/32191045/149277660-368e1efd-6c2f-454f-beb8-09874644f693.png)
BTW, I tried tensorflow_v1 and tensorflow_v2, both failed.

Reply:
Ahh ok, can you add a test in `tests/python/frontend/tflite/test_forward.py` in the relevant spot saying we need to figure out how to make a tflite model with `keep_num_dims` and make a test. 

I will probably accept PR since change is fairly simple.

Reply:
Do you mean add a TODO: in `tests/python/frontend/tflite/test_forward.py` in the relevant spot.

Reply:
yes

Reply:
Can 't fina a suitable line to add the `TODO` and suitable description about the todo.